tests/scripts/anchor-mardown-headings.test.js

```diff
@@ -5,6 +5,8 @@ const test = require('tape')
 test('anchorMarkdownHeadings', (t) => {
   const anchorMarkdownHeadings = require('../../scripts/plugins/anchor-markdown-headings')
 
+  t.timeoutAfter(500)
```

Comment:
Base on what I was able to gather from an [issue logged for tape](https://github.com/substack/tape/issues/178), this forces the timeout to be measured from the time of execution instead of load time. I could be completely wrong.